packages/documentation/src/stories/foundation/icons/icons.styles.scss

```diff
@@ -1,5 +1,107 @@
-.icons {
+@use '@swisspost/design-system-styles/core' as post;
```

Comment:
Why didn't you use bootstrap layout utilities instead of custom CSS? It would have helped with maintainability.

Reply:
I don't wanted to define columns for each and every breakpoint.
Do you think we will need to change this often?

Reply:
It surprised me but you're right, it's not subject to frequent changes, thus it can stay as it is.